Include/object.h

```diff
@@ -78,6 +78,7 @@ whose size is determined when the object is allocated.
 #define PyObject_HEAD                   \
     _PyObject_HEAD_EXTRA                \
     Py_ssize_t ob_refcnt;               \
+    Py_ssize_t str_rstate;                \
```

Comment:
I notice in `stringobject.h` that we use `BSTATE` not `RSTATE`. I wonder if this should be `str_bstate`? I don't feel strongly about it though.